app/views/nodes/show.html.erb

```diff
@@ -34,7 +40,26 @@
       </div>
     </div>
   </div>
-  <div class="tab-pane" id="activity-tab">
+
+  <div class="tab-pane notes <%= "active" if params[:tab] == "notes-tab" %>" id="notes-tab">
+    <div class="inner">
+      <h3>Notes</h3>
+      <div id="note-viewer">
+        <div class="inner note-text-inner">
+          <% if @sorted_notes.any? %>
+            <div id="node-notes-table" class="js-index-table" data-destroy-url="<%= multi_destroy_node_note_path(@node) %>">
+              <%= render 'nodes/notes/toolbar' %>
```

Comment:
will `render 'toolbar'` work here? (also applies to the `render`s below)

Reply:
I can't make it work the other way...